rpmvenv/extensions/core.py

```diff
@@ -85,11 +93,17 @@ def generate(config, spec):
         source = config.core.source
         buildroot = config.core.buildroot
         buildarch = config.core.buildarch
+        requires = config.core.requires
+        provides = config.core.provides
 
         spec.tags['Name'] = name
         spec.tags['Version'] = version
         spec.tags['Release'] = release
         spec.tags['BuildRoot'] = buildroot
+        spec.tags['Requires'] = requires
+        spec.tags['Provides'] = provides
```

Comment:
One more thing to consider is that the `ListOption` will resolve to a list of values. Attaching the list directly to the tags list will cause an issue during rendering because the template expects a primitive value like a string. I would recommend doing something along the lines of:

```python
spec.tags['Provides'] = ', '.join(provides)
```

for these new list values.